shell/src/main/java/alluxio/cli/fs/FileSystemShellUtils.java

```diff
@@ -216,30 +212,15 @@ public static String validatePath(String path) throws IOException {
   }
 
   /**
-   * Gets all supported {@link ShellCommand} classes instances and load them into a map.
+   * Gets all supported {@link Command} classes instances and load them into a map.
```

Comment:
javadoc should point out that this only searches for commands in the same package as `FileSystemShell`